composer.json

```diff
@@ -56,25 +56,27 @@
       "type": "package",
       "package": {
         "name": "alleyinteractive/searchpress",
-        "version": "0.4.1",
+        "version": "dev-main",
         "type": "wordpress-plugin",
         "source": {
           "url": "https://github.com/alleyinteractive/searchpress.git",
           "type": "git",
-          "reference": "v0.4.1"
+          "reference": "main"
         }
       }
     }
   ],
+  "require": {
+    "php": "^8.1 || ^8.2"
+  },
   "require-dev": {
-    "php": "^7.4 || ^8.0",
     "alleyinteractive/alley-coding-standards": "^0.3.0",
-    "alleyinteractive/pest-plugin-wordpress": "^0.4.0",
-    "alleyinteractive/searchpress": "^0.4.1",
+    "alleyinteractive/pest-plugin-wordpress": "^0.5.0",
```

Comment:
Looks like the xml validation is off.

> Your XML configuration validates against a deprecated schema. Migrate your XML configuration using "--migrate-configuration"!

See [here](https://github.com/alleyinteractive/elasticsearch-extensions/actions/runs/7186725055/job/19572761073?pr=65).